.github/fabricbot.json

```diff
@@ -1248,6 +1369,121 @@
       ]
     }
   },
+  {
+    "taskType": "scheduled",
+    "capabilityId": "ScheduledSearch",
+    "subCapability": "ScheduledSearch",
+    "version": "1.1",
+    "config": {
+      "taskName": "Close PRs with no recent activity",
+      "frequency": [
+        {
+          "weekDay": 0,
+          "hours": [
+            0,
+            6,
+            12,
+            18
+          ],
+          "timezoneOffset": 0
+        },
+        {
+          "weekDay": 1,
+          "hours": [
+            0,
+            6,
+            12,
+            18
+          ],
+          "timezoneOffset": 0
+        },
+        {
+          "weekDay": 2,
+          "hours": [
+            0,
+            6,
+            12,
+            18
+          ],
+          "timezoneOffset": 0
+        },
+        {
+          "weekDay": 3,
+          "hours": [
+            0,
+            6,
+            12,
+            18
+          ],
+          "timezoneOffset": 0
+        },
+        {
+          "weekDay": 4,
+          "hours": [
+            0,
+            6,
+            12,
+            18
+          ],
+          "timezoneOffset": 0
+        },
+        {
+          "weekDay": 5,
+          "hours": [
+            0,
+            6,
+            12,
+            18
+          ],
+          "timezoneOffset": 0
+        },
+        {
+          "weekDay": 6,
+          "hours": [
+            0,
+            6,
+            12,
+            18
+          ],
+          "timezoneOffset": 0
+        }
+      ],
+      "searchTerms": [
+        {
+          "name": "isPr",
+          "parameters": {}
+        },
+        {
+          "name": "isOpen",
+          "parameters": {}
+        },
+        {
+          "name": "hasLabel",
+          "parameters": {
+            "label": "no recent activity"
+          }
+        },
+        {
+          "name": "noActivitySince",
+          "parameters": {
+            "days": 14
```

Comment:
I saw you did 14 days for applying the label, and then 14 more days for closing the PR which I think it is reasonable to say if a PR has not had any activity for 28 days it shouldn't be open probably. The only comment here, is in IoT repo we have something very similar except that we apply the label after 21 days of no activity, and then we close the PR if it isn't touched in 7 days. In the end, it is still 28 days, the only difference is that we found that there are a few amount of PRs that don't get touched in 2 weeks, but that is not the case for 3 weeks, so the difference will be that less number of PRs will eventually get the label. This is of course only what happened in IoT, not sure about runtime PRs lifespan so you don't have to change it, just mentioning it as a comment.

Reply:
I'd be open to experiment with various values, this is merely reflecting the numbers we already use in issues. My only feedback on the 21/7 mix is that it leaves users with fewer days to react to the "no recent activity" notice that gets posted in the PR.